lib/milc_interface.cpp

```diff
@@ -1972,7 +2018,10 @@ void milcSetMultigridParam(milcMultigridPack *mg_pack, QudaPrecision host_precis
 
   inv_param.verbosity = verbosity;
 
-  inv_param.verbosity_precondition = verbosity;
+  inv_param.verbosity = input_struct.mg_verbosity[0] ? QUDA_VERBOSE : QUDA_SUMMARIZE;
```

Comment:
This appears that there's no way to enable debug verbosity.  This might be good to add.

Reply:
Good call---I added the full battery of silent/summary/verbose/debug, default summary, so we have it!